File: solutions/python/darts/1/darts.py
def score(x, y):
    scor=0;
    if x**2 + y**2 <= 1:
        scor=10
    elif x**2 + y**2 <= 25:
        scor=5
    elif x**2 + y**2 <= 100:
        scor=1
    else:
        scor=0
    return scor

File: solutions/c/darts/1/darts.c
#include "darts.h"

uint8_t score(coordinate_t pos){
    uint8_t value=0;
    if (pos.x*pos.x + pos.y*pos.y<=1){
        value=10;
    }
    else if (pos.x*pos.x + pos.y*pos.y<=25){
        value=5;
    }
    else if (pos.x*pos.x + pos.y*pos.y<=100){
        value=1;
    }
    return value;
}
